aws-xray/src/awsTest/resources/otel-collector.yml

```diff
@@ -0,0 +1,15 @@
+receivers:
+  awsxray:
```

Comment:
Yup, think this does need to be migrated to an `extension`

Reply:
Is this receiver different than other trace receivers such that it would apply to all or multiple pipelines?

Reply:
It currently is indeed - it opens a separate port which acts as a proxy for sampling requests outside the pipeline. This follows the design of the x-ray daemon but feels weird due to being outside the pipeline